documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,73 @@
-= Introduction
+= devonfw-ide
 
-http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries
-as well as industry proven practices and code conventions.
+== What is devonfw-ide?
+
+`devondw-ide` is a standardized, open-source toolbox or construction kit allowing you to pick and choose those tools which fit the requirements and needs of your project.
+
+It is not another Eclipse. It gives you Eclipse, along with Java, maven, NodeJs, VSCode and everything else you may need to quick start your development. All you need to do is download and extract a zip file and execute its setup, reducing your initial setup time for starting a project from days to minutes.
+
+That is not all. `devonfw-ide` gives you its own command shell where you can execute `devon` commands. Here is a small video which showcases what you can do with `devonfw-ide`:
+
+[youtube video link]
+
+Or you can also try out `devonfw-ide` in our interactive Katacoda scenario without having to download it at all:
+
+[katacoda tutorial link]
+
+http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries as well as industry proven practices and code conventions.
 It massively speeds up development, reduces risks and helps deliver better results.
 
-This document contains the instructions for the tool `devonfw-ide` to set up and maintain your development tools including your favorite IDE (integrated development environment).
+== Why do you need devonfw-ide?
 
-include::features.asciidoc[leveloffset=2]
+Every developer needs great tools to work efficiently. Setting up these tools manually can be tedious and error-prone. Furthermore, some projects may require different versions and configurations of such tools. Especially configurations like code-formatters should be consistent within a project to avoid diff-wars.
```

Comment:
Maybe we should again use this picture of the confluence or wiki article which everyone had to use at least once and that was outdated 2 weeks after writing it. And then you can continue with This shows setting up these tools manually...[your text]

Reply:
Also for the diff-wars use pictures/situations the user knows and wants to avoid

Reply:
Modified accordingly.